apps/lora_app_shell/src/las_cmd.c

```diff
@@ -743,27 +742,21 @@ las_cmd_app_port(int argc, char **argv)
             console_printf("Failed to close app port %u err=%d\n", port, rc);
         }
     } else if (!strcmp(argv[1], "cfg")) {
-        if (argc != 5) {
+        if (argc != 4) {
             console_printf("Invalid # of arguments.\n");
             goto cmd_app_port_err;
         }
-        dr = parse_ull_bounds(argv[2], 0, LORAMAC_TX_MAX_DATARATE, &rc);
-        if (rc) {
-            console_printf("Invalid data rate. Must be between 0 and "
-                           "%d (inclusve)\n", LORAMAC_TX_MAX_DATARATE);
-            return 0;
-        }
         retries = parse_ull_bounds(argv[2], 1, MAX_ACK_RETRIES, &rc);
```

Comment:
I suspect you meant argv[3] here, not argv[2]?

Reply:
Yes indeed. Good catch. I will fix it directly.